wire-compiler/src/main/java/com/squareup/wire/schema/Target.kt

```diff
@@ -399,6 +399,7 @@ data class SwiftTarget(
         val modulePath = context.outDirectory
         val typeName = generator.generatedTypeName(type)
         val swiftFile = SwiftFileSpec.builder(typeName.moduleName, typeName.simpleName)
+          .addImport("Foundation")
```

Comment:
This was needed because of the similar SwiftPoet 1.5 bug around extensions.

Because `init` doesn't explicitly use `Foundation.Data`, it was not getting _implicitly_ imported.
The extension then needs `Foundation.Data` in order to define `unknownFields` which was never imported.

Reply:
If we move to a world where the initializer is minimal, we would run into the same importing problem.